Mathlib/Algebra/BigOperators/Group/Finset/Basic.lean

```diff
@@ -1081,23 +692,11 @@ theorem prod_ite {s : Finset α} {p : α → Prop} {hp : DecidablePred p} (f g :
     ∏ x ∈ s, (if p x then f x else g x) = (∏ x ∈ s with p x, f x) * ∏ x ∈ s with ¬p x, g x := by
   simp [prod_apply_ite _ _ fun x => x]
 
-@[to_additive]
-lemma prod_dite_of_false {p : α → Prop} {_ : DecidablePred p} (h : ∀ i ∈ s, ¬ p i)
-    (f : ∀ i, p i → β) (g : ∀ i, ¬ p i → β) :
-    ∏ i ∈ s, (if hi : p i then f i hi else g i hi) = ∏ i : s, g i.1 (h _ i.2) := by
-  refine prod_bij' (fun x hx => ⟨x, hx⟩) (fun x _ ↦ x) ?_ ?_ ?_ ?_ ?_ <;> aesop
-
 @[to_additive]
 lemma prod_ite_of_false {p : α → Prop} {_ : DecidablePred p} (h : ∀ x ∈ s, ¬p x) (f g : α → β) :
```

Comment:
May I suggest that something is wrong if the `ite` and `dite` versions of the same lemma find themselves in different files?

Reply:
I disagree that this is wrong, but have postponed the `dite` versions into the file with the `ite` versions.

Very happy if someone wants to go through and reduce the import requirements of proofs and then move lemmas upstream, but that's not on the agenda here.

Reply:
What "agenda"? What's the greater context for reducing imports here?

Reply:
"not on the agenda" meaning I'm not interesting in rewriting proofs in this file during this PR.